### PR TITLE
Logic now works correctly and styles are finished

### PR DESCRIPTION
**Pull request by andres-arista**

### CODE REVIEW

File: src/components/__private/dropdownMenu.tsx [hunk 1]
```diff
@@ -22,22 +32,47 @@ const DropdownMenu: React.FC<DropdownMenuProps> = ({
...
+    label,
+    multiple = false,
+    value,
 }) => {
     const theme = useTheme();
+    const isChecked = (value: string[], currentValue: string) => {
+        return value.includes(currentValue);
+    };
     return (
         <React.Fragment>
```
Comment (BrandonDenham) [inline src/components/__private/dropdownMenu.tsx:44]: dont think you need the fragment

File: src/components/dropdown/multipleDropdown.tsx [hunk 2]
```diff
@@ -79,57 +76,47 @@ const MultipleDropdown: React.FC<MultipleDropdownProps> = ({
...
+        (value: string[]) => {
+            const labelElements: string[] = [];
+            value.forEach(currentValue => {
+                children.forEach(currentChildren => {
+                    if (currentValue === currentChildren.value) {
+                        labelElements.push(currentChildren.children);
+                    }
+                });
+            });
+            labelElements.sort();
```
Comment (BrandonDenham) [inline src/components/dropdown/multipleDropdown.tsx:97]: why are we sorting? cant' they order it themselves?

**Review by BrandonDenham — APPROVED** [review]

File: src/components/__private/dropdownMenu.tsx (hunk 1 above)
Comment (andres-arista) [reply src/components/__private/dropdownMenu.tsx:44]: it is complaining if I remove it, I think it is necessary

File: src/components/dropdown/multipleDropdown.tsx (hunk 2 above)
Comment (andres-arista) [reply src/components/dropdown/multipleDropdown.tsx:97]: okay, I will remove the sort, it will be ordered in the order they clicked the checkboxes

**Pull request merged**